src/main/resources/db/migration/changes/Jobs.json

```diff
@@ -0,0 +1,74 @@
+{ "databaseChangeLog": [
+    {
+        "changeSet": {
+          "id": "Jobs-1",
+          "author": "matt (generated)",
+          "preConditions": [
+            {
+              "onFail": "MARK_RAN"
+            },
+            {
+              "not": [
+                {
+                  "tableExists": {
+                    "tableName": "JOBS"
+                  }
+                }
+              ]
+            }
+          ],
+          "changes": [
+            {
+              "createTable": {
+                "columns": [
+                  {
+                    "column": {
+                      "autoIncrement": true,
+                      "constraints": {
+                        "primaryKey": true,
+                        "primaryKeyName": "CONSTRAINT_2"
```

Comment:
@MPriston I think I agree with @scottpchow23 's suggestion that we need a naming convention for these (unless I'm misunderstanding something).

What would be a suitable convention?

Reply:
And what is the scope where these have to be unique?  Just the one database table? The entire database?